lib/tasks/bulk_update_csv.rake

```diff
@@ -83,21 +83,78 @@ end
 
 def overwrite_multivalue_row(logger, work, row, property)
   to_value = row[:to]&.to_s
-  work[row[:property]] = to_value.blank? ? nil : [to_value&.split('|')].flatten
+  return work if to_value.blank?
+
+  if ordered_property? row[:property]
+    values = case row[:property]
+    when 'title'
+      work.nested_ordered_title_attributes = overwrite_ordered_property_value(work, 'ordered_titles', 'title', to_value)
+    when 'creator'
+      work.nested_ordered_creator_attributes = overwrite_ordered_property_value(work, 'ordered_creators', 'creator', to_value)
+    when 'abstract'
+      work.nested_ordered_abstract_attributes = overwrite_ordered_property_value(work, 'ordered_abstracts', 'creator', to_value)
```

Comment:
I think the `value_field` argument should be `abstract` for this case right? It looks like it should be:
```
work.nested_ordered_abstract_attributes = overwrite_ordered_property_value(work, 'ordered_abstracts', 'abstract', to_value)
```